jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -169,6 +169,14 @@ public final void setMaxMultiplex(int maxMultiplex)
         if (maxMultiplex < 1)
             throw new IllegalArgumentException("Max multiplex must be >= 1");
         this.maxMultiplex = maxMultiplex;
+
+        try (Locker.Lock l = locker.lock())
+        {
+            if (closed)
+                return;
+
+            entries.forEach(entry -> entry.setMaxMultiplex(maxMultiplex));
```

Comment:
I removed this.

It was there because the first request of a new connection is first reserving an entry, which would inherit the initial `maxMultiplex=1024`. However, when the connection is established, the SETTINGS frame with `max_concurrent_streams` sets `Pool.maxMultiplex`, but did not change that of the reserved entry, so I thought this was a good place to set it.
However, a better place to do it is in `AbstractConnectionPool.FutureConnection.succeeded()` when we `enable()` the reserved `Entry`.